x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -164,15 +162,6 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
             )
           )
           .join('\n');
-        /*
-         * Custom recovery actions aren't yet available in the alerting framework
-         * Uncomment the code below once they've been implemented
-         * Reference: https://github.com/elastic/kibana/issues/87048
-         */
-        // } else if (nextState === AlertStates.OK && prevState?.alertState === AlertStates.ALERT) {
-        // reason = results
-        //   .map((result) => buildReasonWithVerboseMetricName(group, result[group], buildRecoveredAlertReason))
```

Comment:
@simianhacker Do we plan to have reason messages for recovered alerts? Not as part of this PR, but shall we create a ticket for this?